spec/use_packwerk_spec.rb

```diff
@@ -1,6 +1,10 @@
 # typed: false
 RSpec.describe UsePackwerk do
+
```

Comment:
Lots of new tests for the new behavior. Definitely view with "hide whitespace change" set to true since I did a lot of context nesting of the old tests.